mix.exs

```diff
@@ -50,9 +50,10 @@ defmodule UUID.Mixfile do
   # List of dependencies.
   defp deps do
     [
+      {:ecto, "~> 3.5", only: [:dev, :test]},
```

Comment:
We should make it an optional dependency and since we already guard against ParameterizedTypes not being there, we can use a lower requirement (I haven't tested it but it should work on 3.0.0)

```suggestion
      {:ecto, "~> 3.0", optional: true}
```